indico/modules/events/client/js/creation.js

```diff
@@ -163,42 +163,42 @@ import camelizeKeys from 'indico/utils/camelize';
                 success(data) {
                     data = camelizeKeys(data);
                     if (data.userBooking) {
-                        $createBooking.val(false);
+                        $createBooking.val('false');
                         $currentMessage = $userBookingMessage;
                         addCalendarLink($currentMessage);
                     } else if (data.userPrebooking) {
-                        $createBooking.val(false);
+                        $createBooking.val('false');
                         $currentMessage = $userPrebookingMessage;
                         addCalendarLink($currentMessage);
                     } else if (data.conflictBooking) {
-                        $createBooking.val(false);
+                        $createBooking.val('false');
                         $currentMessage = $conflictBookingMessage;
                         addCalendarLink($currentMessage);
                     } else if (data.unbookable) {
-                        $createBooking.val(false);
+                        $createBooking.val('false');
                         $currentMessage = $unbookableMessage;
                         addCalendarLink($currentMessage);
                     } else if (data.conflictPrebooking) {
                         if (data.canBook) {
-                            $createBooking.val($bookingSwitchPrebooking[0].checked);
+                            $createBooking.val(`${$bookingSwitchPrebooking[0].checked}`);
                             $currentMessage = $conflictPrebookingMessage;
                             addCalendarLink($currentMessage);
                         } else if (data.canPrebook) {
-                            $createBooking.val($prebookingSwitchPrebooking[0].checked);
+                            $createBooking.val(`${$prebookingSwitchPrebooking[0].checked}`);
```

Comment:
I'm not even mad, that's creative!

But I'd use one of these instead:

- `$prebookingSwitchPrebooking[0].checked.toString()`
- `String($prebookingSwitchPrebooking[0].checked)`